modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/ags_linear_solver.h

```diff
@@ -12,55 +12,38 @@ namespace lbs
 {
 
 /**Linear Solver specialization for Within GroupSet (WGS) solves.*/
-class AGSLinearSolver : public LinearSolver
+class AGSLinearSolver
 {
+private:
+  std::shared_ptr<LinearSolverContext> context_ptr_;
+  int maximum_iterations_;
+  double tolerance_;
+  bool verbose_;
+
 public:
   /**Constructor.
-   * \param iterative_method string Across Groupset iterative method.
    * \param ags_context_ptr Pointer Pointer to the context to use.
-   * \param groupspan_first_id int First group index.
-   * \param groupspan_last_id int Last group index.
    * \param verbose bool Flag to enable verbose output.*/
-  AGSLinearSolver(std::string iterative_method,
-                  std::shared_ptr<AGSContext> ags_context_ptr,
-                  int groupspan_first_id,
-                  int groupspan_last_id,
-                  bool verbose = true)
-    : LinearSolver(std::move(iterative_method), ags_context_ptr),
-      groupspan_first_id_(groupspan_first_id),
-      groupspan_last_id_(groupspan_last_id),
-      verbose_(verbose)
+  AGSLinearSolver(std::shared_ptr<AGSContext> ags_context_ptr)
+    : context_ptr_(ags_context_ptr), maximum_iterations_(1), tolerance_(1.0e-6), verbose_(false)
   {
   }
 
-  ~AGSLinearSolver() override;
-
-  int GroupSpanFirstID() const { return groupspan_first_id_; }
+  ~AGSLinearSolver() {}
 
-  int GroupSpanLastID() const { return groupspan_last_id_; }
+  void Solve();
 
   bool IsVerbose() const { return verbose_; }
 
-  void SetVerbosity(bool verbose_y_n) { verbose_ = verbose_y_n; }
-
-  void Solve() override;
-
-protected:
-  void SetSystemSize() override;
+  void SetVerbosity(bool verbose) { verbose_ = verbose; }
 
-  void SetSystem() override;
+  double GetTolerance() { return tolerance_; }
```

Comment:
Curious to know everyone's thoughts on `GetTolerance` vs. `Tolerance` for getters broadly.

Reply:
Personally, I've always preferred them without Get/Set. In this case, I was just following what was already there.